modin/pandas/base.py

```diff
@@ -3245,7 +3355,7 @@ def __getitem__(self, key):
             and (not hasattr(self, "columns") or key not in self.columns)
         ):
             indexer = convert_to_index_sliceable(
-                getattr(pandas, type(self).__name__)(index=self.index), key
+                getattr(pandas, "DataFrame")(index=self.index), key
```

Comment:
Why this changed?

Reply:
Because now pandas passes `DataFrame` object only. If `Series` is passed to the function, an exception may be arisen when looping over columns.